build/vsts-ci.yml

```diff
@@ -297,14 +297,15 @@ jobs:
     DOTNET_CLI_TELEMETRY_OPTOUT: 1
     DOTNET_SKIP_FIRST_TIME_EXPERIENCE: 1
     DOTNET_MULTILEVEL_LOOKUP: 0
+    NUGET_PACKAGES: $(Build.SourcesDirectory)/.packages
```

Comment:
Why is this only necessary here?

Reply:
This gets set automatically by arcade during the build phase, so usually nothing is needed. In the final step of our official build though we build/pack, sign, and upload symbols in different steps. Because they happen in different steps the value is incorrect during all steps except our build/pack.